src/main/java/ml/shifu/shifu/core/dtrain/wdl/WDLParallelGradient.java

```diff
@@ -89,14 +91,11 @@ public WDLParallelGradient(final WideAndDeep wnd, int threadNumber, ConcurrentMa
      * So to achieve this, we need to adjust the start index of training set for each iteration. And also need update
      * the slice for each thread.
      *
-     * @param start
-     *          The start index in {@link WDLParallelGradient#trainData}, this iteration to train start with.
      * @param miniBatchSize
      *          The mini-batch size for each iteration.
      */
-    private void adjustTrainSet(int start, int miniBatchSize) {
-        assert start >= 0;
-        int recordCount = start + miniBatchSize > this.trainData.size() ? this.trainData.size() - start: miniBatchSize;
+    private void adjustTrainSet(int miniBatchSize) {
```

Comment:
Why we need to change this method? The new method's behavior is the same with `start = 0`.
I also find that the logic of `adjustTrainSet` and `adjustTestSet` is very similar. If we change one, we also need to change the other. If we can extract a common method for these two, it should be better.